docs/docs/kds-rest-api.md

```diff
@@ -33,11 +33,12 @@ a cryptographic link between the client and the enclave.
 }
 ```
 
-| Field | Description |
-| ----- | ----------- |
-| name | A name for the key. The name is used during key derivation so can be used to ensure that keys with the same master key and constraint configuration can be uniquely generated. |
-| masterKeyType | The type of master key provider that is used to source the master key. |
-| policyConstraint | The constraint policy to apply to the key. A key will only be released to an enclave that meets these constraints. |
+| Field | Description                                                                                                              |
+| ----- |--------------------------------------------------------------------------------------------------------------------------|
+| name | A name for the key. The KDS uses the name field to generate unique keys with the same master key and constraint configuration. |
+| masterKeyType | The type of master key provider the KDS should use to source the master key.                                          |
```

Comment:
Link to KDS master key types documentation (not entirely sure where this is!) maybe kds-detail?